src/diffusers/models/attention_processor.py

```diff
@@ -697,27 +699,32 @@ def norm_encoder_hidden_states(self, encoder_hidden_states: torch.Tensor) -> tor
 
     @torch.no_grad()
     def fuse_projections(self, fuse=True):
-        is_cross_attention = self.cross_attention_dim != self.query_dim
         device = self.to_q.weight.data.device
         dtype = self.to_q.weight.data.dtype
 
-        if not is_cross_attention:
+        if not self.is_cross_attention:
             # fetch weight matrices.
             concatenated_weights = torch.cat([self.to_q.weight.data, self.to_k.weight.data, self.to_v.weight.data])
             in_features = concatenated_weights.shape[1]
             out_features = concatenated_weights.shape[0]
 
             # create a new single projection layer and copy over the weights.
-            self.to_qkv = self.linear_cls(in_features, out_features, bias=False, device=device, dtype=dtype)
+            self.to_qkv = self.linear_cls(in_features, out_features, bias=self.use_bias, device=device, dtype=dtype)
```

Comment:
I think it's safe to always make it to False because attention layers don't use bias.

Reply:
Then why Attention block need a bias param when init?
And it actually make test fails in some case where `attention_bias==True`.

Reply:
You mean fused projections make some of the tests fail when attention bias is true?

Reply:
Yes, the results will no longer be same when bias is enabled in original Attention but fused projections do not have bias

Reply:
As mentioned by @yiyixuxu as well if we decide to not add projection fusion to the other models, then we don't have this case anymore. Am I right?